armi/reactor/composites.py

```diff
@@ -3118,6 +3118,18 @@ def getBoundingCircleOuterDiameter(self, Tc=None, cold=False):
         getter = operator.methodcaller("getBoundingCircleOuterDiameter", Tc, cold)
         return sum(map(getter, self))
 
+    def getPuMoles(self):
```

Comment:
Why are we moving this?

I mean, I don't like all these single-element/nuclide methods.

I think we should just have one method:

`Composite.getZMoles(self, z)`

Or some such.

Reply:
We want to evaluate this on both blocks and components now. In this PR It gets called within iblocks.py::Block::completeInitialLoading on both Blocks and Components so I moved the tree.